ETBezierHandle.m

```diff
@@ -79,18 +83,22 @@ - (id) initWithActionHandler: (ETActionHandler *)anHandler
 			case NSCurveToBezierPathElement:
 				[handles addObject: AUTORELEASE([[ETBezierHandle alloc] initWithActionHandler: [ETBezierPointActionHandler sharedInstance]
 				                                                            manipulatedObject: aTarget
+																		   objectGraphContext: (COObjectGraphContext *)aContext
```

Comment:
Same indentation issue here. Indenting ETBezierHandler calls with spaces instead of tabs would be better.